examples/rename_brainvision_files.py

```diff
@@ -78,4 +78,4 @@
 #
 # Lastly, there is a tool to check the integrity of your BrainVision files.
 # For that, see the
-# `BrainVision Validator <https://github.com/sappelhoff/brainvision-validator>`_
+# `BrainVision Validator <https://github.com/sappelhoff/brainvision-validator>`
```

Comment:
are you sure this works after removing the underscore?

Reply:
no, thanks for the catch ... I thought it was trailing whitespace O.o